src/sentry/models/eventattachment.py

```diff
@@ -10,6 +10,7 @@ class EventAttachment(Model):
     __core__ = False
 
     project_id = BoundedBigIntegerField()
+    group = FlexibleForeignKey("sentry.Group", null=True)
```

Comment:
Why does it have to be nullable ? Is it only for the migration (in that case you can just give it a default value) or is it for issueless events ?

Reply:
Can issueless events have attachments at all?
We will need to fetch the group ID from Snuba and in the case that it somehow fails or is not there yet, we probably want to still write the attachment without a group ID.

Reply:
It has to be null for adding new columns.